trade.md

```diff
@@ -42,7 +42,32 @@ For each traded commodity with the logit-based Armington approach, this structur
 1. The global "traded commodity" is supplied by gross exports from each of GCAM's geopolitical regions.
 2. The "regional commodity" sector within each region allocates market share to domestic production versus imports of the global "traded commodity".
 
-The logit parameters are calibrated based on literature information of trade elasticities. For example, (1) livestock products have generally higher logit exponents (Hertel et al., 2007), and (2) the regional logit exponent and interantional logit exponent are tied with "rule of two" (Liu et al., 2004).
+The logit parameters are calibrated based on literature information of Armington trade elasticities (Table 1). For example, (1) livestock products have generally higher logit exponents (Hertel et al., 2007), and (2) the regional logit exponent and international logit exponent are tied with "rule of two" (Liu et al., 2004). 
+
+In addition, bilateral trade is not a default option in GCAM currently. However, there is a working branch of GCAM that enables bilateral trade modeling of agricultural products by further disaggregating international "traded sector" to regional "traded sector" following the Armington framework. 
```

Comment:
Xin,

Pralit reminded me that we should just be very clear in this documentation about what is actually in the model now and not mention the bilateral trade.  I'm tempted to mention it myself because the structure does allow it.  But I think it's best to not confuse and just to remove those two lines.

Reply:
Sure. That makes sense! Will remove these lines. And thank you for the edits!